runtime/src/main/cpp/TypeInfo.h

```diff
@@ -54,7 +54,8 @@ enum Konan_RuntimeType {
 enum Konan_TypeFlags {
   TF_IMMUTABLE = 1 << 0,
   TF_ACYCLIC   = 1 << 1,
-  TF_INTERFACE = 1 << 2
+  TF_INTERFACE = 1 << 2,
+  TF_NON_CLONEABLE = 1 << 3
```

Comment:
Why not `TF_CLONEABLE`?
`TF_NON_CLONEABLE` looks somewhat confusing, especially with opt-in approach.